core/templates/dev/head/components/state-directives/outcome-editor/outcome-editor.directive.ts

```diff
@@ -17,6 +17,9 @@
  */
 
 require('domain/utilities/UrlInterpolationService.ts');
+require(
+  'components/state-directives/outcome-editor/' +
+  'outcome-destination-editor.directive.ts');
```

Comment:
This should be in a different section, also requires for `outcome-feedback-editor` and `angular-html-bind` are missing.

Reply:
Done